### DISCUSSION TITLE
negspacy not working

### DISCUSSION THREAD

**Discussion by cangainuren** [explosion/spaCy#11252] [Help: Coding & Implementations]
I'm having problem running the following example. I have spacy 3.4 and latest negspacy.
 
import spacy
from negspacy.negation import Negex

nlp = spacy.load("en_core_web_sm")
nlp.add_pipe("negex", config={"ent_types":["PERSON","ORG"]})

ValueError: [E002] Can't find factory for 'negex' for language English (en). This usually happens when spaCy calls `nlp.create_pipe` with a custom component name that's not registered on the current language class. If you're using a Transformer, make sure to install 'spacy-transformers'. If you're using a custom component, make sure you've added the decorator [`@Language.component`](mailto:%60@Language.component%60) (for function components) or [`@Language.factory`](mailto:%60@Language.factory%60) (for class components).

**Comment by polm** [comment]
Your code works fine for me - can you confirm you have the latest version of negspaCy installed? Check the output of `pip freeze`. 

Also, note that negspaCy is a third party project not developed by us, and it may be better to ask questions at [their repo](https://github.com/jenojp/negspacy).
